### PR TITLE
blog: Postman Add-on

### PR DESCRIPTION
**Pull request by VitikaSoni**

### CODE REVIEW

File: site/content/blog/2023-09-23-postman-add-on/index.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+title: "Postman Add-on"
+summary: "Import Postman collections with the new Postman add-on for ZAP."
+type: post
+tags:
+- blog
+- postman
+- gsoc
+- gsoc-2023
+date: "2023-09-23"
```
Comment (psiinon) [inline site/content/blog/2023-09-23-postman-add-on/index.md:10]: Lets publish this on Monday 25th - this gives more time for reviews and we get more engagement on weekdays.

File: site/content/blog/2023-09-23-postman-add-on/index.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+authors:
+    - vitikasoni
+---
+This extension allows us to easily import [Postman collections](https://www.postman.com/collection/) into ZAP, and hence we can then use ZAP’s powerful security testing capabilities to identify potential vulnerabilities in our APIs developed using Postman.
```
Comment (psiinon) [inline site/content/blog/2023-09-23-postman-add-on/index.md:14]: Start by linking to the help for your extension (which will be available as soon as we publish it) and tell people how to install it (via the Marketplace https://www.zaproxy.org/addons/ )

File: site/content/blog/2023-09-23-postman-add-on/index.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+- Enable importing through API and command line.
+- Enable the import of variables defined in the collection definition.
+- Introduce automation support, and more.
+
+Stay tuned for these upcoming enhancements to our addon! Checkout the [Community page](https://www.zaproxy.org/community/) to get in touch.
+
+## My GSoC Journey and Thanks:
+
+The development of this add-on was part of my Google Summer of Code (GSoC) adventure (the project page can be found [here](https://summerofcode.withgoogle.com/archive/2023/projects/OlBxaE5X)).
```
Comment (psiinon) [inline site/content/blog/2023-09-23-postman-add-on/index.md:43]: link to https://summerofcode.withgoogle.com/ ?

File: site/content/blog/2023-09-23-postman-add-on/index.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+My mentors - Simon, Ricardo and Rick were there, guiding me every step of the way. With every discussion and code review with them, there was a lot of learning for me. I am truly thankful to them. 😄
```
Comment (psiinon) [inline site/content/blog/2023-09-23-postman-add-on/index.md:44]: Link to our team pages?

**Review by psiinon — COMMENTED** [review]
Looking good!
Just a few minor comments..

**Comment by psiinon** [comment]
I'll see if I can come up with an image which incorporates ZAPbot and the Postman icon 😁

**Comment by VitikaSoni** [comment]
Made the requested changes :)

**Comment by psiinon** [comment]
How about this as the social media pic?

blog-postman

**Comment by psiinon** [comment]
Scratch that 😦 https://www.postman.com/legal/logo-usage/

**Comment by psiinon** [comment]
But theres this kind of thing...

blog-postman-4

**Comment by psiinon** [comment]
tidied up a bit..

blog-postman-4

File: site/content/blog/2023-09-25-postman-add-on/index.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+- gsoc-2023
+date: "2023-09-25"
+authors:
+    - vitikasoni
+---
+This extension allows us to easily import [Postman collections](https://www.postman.com/collection/) into ZAP, and hence we can then use ZAP’s powerful security testing capabilities to identify potential vulnerabilities in our APIs developed using Postman.
+
+## Installing the add-on
+
+The help page of the add-on can be found [here](https://www.zaproxy.org/docs/desktop/addons/postman-support/) and you can install it via the [ZAP Marketplace](https://www.zaproxy.org/addons/).
```
Comment (thc202) [inline site/content/blog/2023-09-25-postman-add-on/index.md:18]: The scheme and domain (`https://www.zaproxy.org`) should be removed, leave just the full path.

File: site/content/blog/2023-09-25-postman-add-on/index.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
+---
+title: "Postman Add-on"
+summary: "Import Postman collections with the new Postman add-on for ZAP."
+images:
+- /blog/2023-09-25-postman-add-on/images/zapbot-postman.png
```
Comment (psiinon) [inline site/content/blog/2023-09-25-postman-add-on/index.md:5]: This needs to have the full URL or it wont work as a social preview image

Comment (VitikaSoni) [reply site/content/blog/2023-09-25-postman-add-on/index.md]: Done

**Review by psiinon — APPROVED** [review]

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**